MVVM/ViewModel/MainViewModel.cs

```diff
@@ -81,6 +81,7 @@ public override void _Ready()
 			ManagePtzStatus();
 			ManageRtspStatus();
 
+			_rtspClient.FrameReceived += IsRtspOkWrapper;
```

Comment:
I'm not sure if it's the best place for a line like this, but it works. Putting it in the same place as others makes the app crash, because of _rtspClient not being inicialised yet.

Reply:
*It works some times..
It should be inside `void ManageRtspStatus()` method like this:
```cs
		private void ManageRtspStatus()
		{
			switch (LocalSettings.Singleton.Camera.EnableRtspStream)
			{
				case true when _rtspClient is null:
					_rtspClient = new();
					_rtspClientWeak = new(_rtspClient);
					_rtspClient.FrameReceived += IsRtspOkWrapper; // <--- subscribe to event after rtspClient is created.
					break;
				case false when _rtspClient is not null:
					_rtspClient.FrameReceived -= IsRtspOkWrapper; // <--- unsubscribe from event before rtspClient is disposed.
					_rtspClient.Dispose();
					_rtspClient = null;
					break;
			}
		}
```

Reply:
Resolved in [3379ebc](https://github.com/wisniax/RoverControlApp/pull/19/commits/3379ebc6588b289d522a38f2835cb88f8962ed78)

Reply:
Please unsubscribe 
```cs
_rtspClient.FrameReceived -= OnRtspFrameReceivedWrapper; // <--- unsubscribe from event before rtspClient is disposed.
```

Reply:
Didn't notice, resolved in [b814716](https://github.com/wisniax/RoverControlApp/pull/19/commits/b814716555d3c4addf27a7c4b604bb211a45ae6c)